app/views/service_providers/_form.html.erb

```diff
@@ -344,64 +344,48 @@ B7xxt8BVc69rHV15A0qyx77CLSj3tCx2IUXVqRs5mlvA==
+          <br><legend><span class="text-heavy">Sign-in help text:</span></legend>
+          <%= s.input :en, label: "English", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_in", "en"), readonly: readonly_help_text? } %>
+          <%= s.input :es, label: "Español", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_in", "es"), readonly: readonly_help_text?  } %>
+          <%= s.input :fr, label: "Français", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_in", "fr"), readonly: readonly_help_text? } %>
         <% end %>
 
         <%= h.fields_for :sign_up do |s| %>
           <br><legend><b>Sign-up help text:</b></legend>
-          <%= s.input :en, label: "English", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_up", "en") } %>
-          <%= s.input :es, label: "Español", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_up", "es") } %>
-          <%= s.input :fr, label: "Français", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_up", "fr") } %>
+          <%= s.input :en, label: "English", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_up", "en"),readonly: readonly_help_text? } %>
+          <%= s.input :es, label: "Español", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_up", "es"), readonly: readonly_help_text? } %>
+          <%= s.input :fr, label: "Français", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_up", "fr"), readonly: readonly_help_text? } %>
         <% end %>
 
         <%= h.fields_for :forgot_password do |f| %>
           <br><legend><b>Forgot password help text:</b></legend>
-          <%= f.input  :en, label: "English", as: :text, required: false, input_html: { value: service_provider.help_text.dig("forgot_password", "en") } %>
-          <%= f.input :es, label: "Español", as: :text, required: false, input_html: { value: service_provider.help_text.dig("forgot_password", "es") } %>
-          <%= f.input :fr, label: "Français", as: :text, required: false, input_html: { value: service_provider.help_text.dig("forgot_password", "fr") } %>
+          <%= f.input  :en, label: "English", as: :text, required: false, input_html: { value: service_provider.help_text.dig("forgot_password", "en"), readonly: readonly_help_text? } %>
+          <%= f.input :es, label: "Español", as: :text, required: false, input_html: { value: service_provider.help_text.dig("forgot_password", "es"), readonly: readonly_help_text? } %>
+          <%= f.input :fr, label: "Français", as: :text, required: false, input_html: { value: service_provider.help_text.dig("forgot_password", "fr"), readonly: readonly_help_text? } %>
         <% end %>
-    <% end %>
     </fieldset>
-  <% else %>
-    <% if @help_text_empty %>
```

Comment:
i removed this big duplicative conditional which had a lot of invalid html, and instead made the `readonly` a conditional